application.properties

```diff
@@ -0,0 +1,2 @@
+db.name=file
```

Comment:
Why you have 7 homeworks in pull request to 1 homework ?

Reply:
@Siril-O I did it by chance, when I close the pool request, everything will be fine